miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditProjectController.java

```diff
@@ -283,11 +283,6 @@ public ModelAndView setupForm(@PathVariable Long projectId, ModelMap model) thro
     model.put("overviews", project.getOverviews());
 
     Map<Long, String> overviewMap = new HashMap<>();
-    for (ProjectOverview po : project.getOverviews()) {
-      if (po.getWatchers().contains(user)) {
-        overviewMap.put(po.getId(), user.getLoginName());
-      }
-    }
     model.put("overviewMap", overviewMap);
```

Comment:
remove map declaration and model attribute

Reply:
This happens in #1782. I didn't want to have to edit the JSP much at this point.